README.md

```diff
@@ -23,18 +39,70 @@ In order to use this pipeline, you will also have to install the [RGI][], [stara
 | staramr | `0.4.0` | <https://toolshed.g2.bx.psu.edu/view/nml/staramr/930893c83a1a> |
 | shovill | `1.0.4` | <https://toolshed.g2.bx.psu.edu/view/iuc/shovill/865119fcb694> |
 
-# Screenshots
+# Tutorial
 
-## Pipeline
+This tutorial shows how to run the **AMR Detection** pipeline in IRIDA.
+
+## Tutorial Data
+
+The pipeline requires as input paired-end sequence reads in FASTQ format. We will be using Illumina MiSeq data from a sample of **Campylobacter jejuni** for this tutorial with run ID `SRR8914694`. Paired-end fastq files can be downloaded from <https://www.ebi.ac.uk/ena/data/view/SRR8914694> (the **FASTQ files (FTP)** links). A collection additional example data can be found in the [IRIDA Sample Data][] package.
```

Comment:
Add the word `of` after the word `collection` in the last sentence please

Reply:
Done in f3744f7e7f81ccb10c32697136e0bd9322077ced